libs/pxweb2-ui/src/lib/components/Typography/BodyLong/BodyLong.tsx

```diff
@@ -0,0 +1,40 @@
+import cl from 'clsx';
+import classes from './BodyLong.module.scss';
+
+export interface BodyLongProps
+  extends React.HTMLAttributes<HTMLParagraphElement> {
+  size?: 'medium' | 'small';
+  spacing?: boolean;
+  align?: 'start' | 'center' | 'end';
+  color?: 'default' | 'subtle';
+  weight?: 'regular' | 'bold';
+  children: React.ReactNode;
+}
+
+export function BodyLong({
+  children,
+  size = 'medium',
+  align = 'start',
+  color = 'default',
+  weight = 'regular',
+  spacing = false,
+  ...rest
+}: BodyLongProps) {
+  return (
+    <p
```

Comment:
Consider using <span> instead, a <p> tag could be limiting in the future. We can always place the component within a <p> tag if we'd like to use a <p> tag. 

<p><BodyLong>text</BodyLong></p>

The paragraph tag has semantical meaning, and we might not want to use a <p> tag everywhere we want to use bodylong.

Reply:
The `<p>` tag also adds some margin we might not always want

Reply:
Discussed on teams, resolved.